notebooks/Training_a_LoRA_With_Instruct_Lab.ipynb

```diff
@@ -1,931 +1,953 @@
 {
-  "cells": [
```

Comment:
What is this change about?

Reply:
this was just changed by ruff format command